docs/topics/addons.rst

```diff
@@ -0,0 +1,190 @@
+.. _topics-addons:
+
+=======
+Add-ons
+=======
+
+Scrapy's add-on system is a framework which unifies managing and configuring
+components that extend Scrapy's core functionality, such as middlewares,
+extensions, or pipelines. It provides users with a plug-and-play experience in
+Scrapy extension management, and grants extensive configuration control to
+developers.
+
+
+Activating and configuring add-ons
+==================================
+
+Add-ons and their configuration live in Scrapy's
+:class:`~scrapy.addons.AddonManager`. During a :class:`~scrapy.crawler.Crawler`
+initialization the add-on manager will read a list of enabled add-ons from your
+``ADDONS`` setting.
```

Comment:
I wonder if we should remove AddonManager here. We don't put emphasis on e.g. MiddlewareManager, as users are not goint to use this directly. 

```suggestion
During a :class:`~scrapy.crawler.Crawler`
initialization it reads a list of enabled add-ons from your
``ADDONS`` setting.
```

Reply:
I agree with removing the mention, although I would suggest a slightly different wording:

```suggestion
During :class:`~scrapy.crawler.Crawler` initialization, the list of enabled 
add-ons is read from your ``ADDONS`` setting.
```